rdkPlugins/Storage/README.md

```diff
@@ -112,3 +136,26 @@ For every dynamic mount point the Storage plugin should create, there should be
 }
 ```
 
+### Creating mount owners
+For every mount owner point the Storage plugin should create, there should be one item in the array of "mountOwner". The options inside this object goes as follows:
```

Comment:
For `mountOwner` Storage plugin doesn't create anything. It just changes ownership. Please say that this WILL change ownership in the host, and that path are resolved to the host. Plus it would be nice to mention that similar functionality is done in Podman https://github.com/containers/podman/commit/874f2327e6ca963edda7cc46819d51048d3d19a8 so it is no total heresy to do such thing ;)